source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,56 @@
+- Apps cannot access databases created for the AP users (these are all of the "curated" databases created by the Data Engineering Team)
+- Users can access read/write to app buckets (permissions to do this are given via the Control Panel).
+
+So if you create an app that needs specific data from our databases then you need to consider the following practices:
+
+#### Data Minimisation
+
+When writing data to your app bucket you should only ever give the app the minimal data that it needs. For example, even though your analysis might require personal or low level data, your app may well be presenting this data at an aggregated level, in which case you should put just the aggregated data into the app bucket. You should never put data in the app bucket that you do not want users to see (even if the code behind the app itself has no method to expose that data to the user via the app).
+
+#### Writing data to an App bucket
+
+There are multiple ways you can do this and it is up to you to determine how best to do it.
+
+- You could manually run some code from your RStudio or JupyterLab environment that writes the necessary data to the app bucket whenever it needs updating
+- You could write an Airflow script that writes data from a database to the app bucket meaning the app still only has access to it's very specific area in AWS.
+- When updating data for Rshiny apps you often need to restart the R session so it refreshes the cached data. You can do this using Airflow (to see how to do this search the [airflow dags repository](https://github.com/moj-analytical-services/airflow-dags) for the `BashOperator`).
+
+#### Creating a database for your app
+
+> Note this is only recommended for users who understand how to create/manage databases on Athena. See other parts of guidance on these tools for more details on how to use them.
+
+In most cases you will not need to do this as it will be unnecessary. However, if you feel like you do need to create an Athena SQL database because your app requires the processing power from Athena or the data you are reading in is too large to be cached then you will need to do the following:
+
+- Create an Athena database that will be used for your app (the data will still sit in the app bucket but it will also need a Glue schema created in order for Athena to query it).
+- Message on the data-engineer slack channel to ask them to add this database for your app. As well as explaining why the Athena database is needed for the app.
+
+Your app role will then be given access to only read the specific database for your app and no other databases. This again restricts what an app can access, compared to the wider access given to AP users. If you require to update the app database follow the same tips / routes to writing data in S3 in the ["writing data to an app bucket" section](#writing-data-to-an-app-bucket).
```

Comment:
I don't like this "If you require to update", it sounds like a translation from Spanish. Is this about the app writing to the database during use, or is it about updates to the data from upstream?